.github/workflows/tests.yml

```diff
@@ -27,6 +27,27 @@ jobs:
                 command: composer install
             - name: Run Script
               run: vendor/bin/phpunit
+    test_windows_with_com:
+        runs-on: windows-latest
+        strategy:
+            matrix:
+                php: [ "8.0", "8.1", "8.2", "8.3" ]
+        name: PHP ${{ matrix.php }} Windows with COM Unit Test
+        steps:
+            - uses: actions/checkout@v4
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: ${{ matrix.php }}
+                extensions: gmp, php_com_dotnet
+            - name: Install Dependencies
+              uses: nick-invision/retry@v3
+              with:
+                timeout_minutes: 10
+                max_attempts: 3
+                command: composer install
+            - name: Run Script
+              run: vendor/bin/phpunit --filter GCECredentialsTest
```

Comment:
Instead of adding a new job, add "windows-latest" and the extensions to an `includes` section of the matrix for the existing job. It will look something like this instead:

```yaml
        strategy:
            matrix:
                php: [ "8.0", "8.1", "8.2", "8.3" ]
                os:   [ ubuntu-latest ]
                include:
                  - os: windows-latest
                    php: "8.1"
        name: PHP ${{matrix.php }} Unit Test${{ matrix.os == 'windows-latest' && ' on Windows' || '' }}
        runs-on: ${{ matrix.os }}
        steps:
            - uses: actions/checkout@v4
            - name: Setup PHP
              uses: shivammathur/setup-php@v2
              with:
                php-version: ${{ matrix.php }}
                extensions: ${{ matrix.os == 'windows-latest' && 'gmp, php_com_dotnet' || '' }}
```